modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-channel-impl/src/main/java/com/liferay/headless/commerce/admin/channel/internal/dto/v1_0/converter/CategoryDisplayPageDTOConverter.java

```diff
@@ -18,7 +18,7 @@
  * @author Danny Situ
  */
 @Component(
-	property = "dto.class.name=com.liferay.commerce.product.model.CPDisplayLayout-Category",
+	property = "dto.class.name=CategoryDisplayPage",
```

Comment:
I meant the fully qualified class name, sorry for the confusion. Fully qualified class name is better because you can get the version while using the registry.